betting-game-react/src/components/Navbar/NavbarButtonProps.tsx

```diff
@@ -0,0 +1,25 @@
+import { ABOUTUS, MATCHES, STANDINGS } from '../../config/routes';
+
+export interface NavbarButtonProps {
+  buttonText: string;
+  buttonURL: string;
+  buttonTestIdPrefix: string;
+}
+
+export const navbarButtons: NavbarButtonProps[] = [
+  {
+    buttonURL: MATCHES.path,
+    buttonText: 'Matches',
+    buttonTestIdPrefix: 'Matches',
+  },
+  {
+    buttonURL: STANDINGS.path,
+    buttonText: 'Standings',
+    buttonTestIdPrefix: 'Standings',
+  },
+  {
+    buttonURL: ABOUTUS.path,
+    buttonText: 'About us',
+    buttonTestIdPrefix: 'AboutUs',
+  },
+];
```

Comment:
I prefer having data and prop dependencies within the component that need them.
I think this is one of these things where we'll discuss and find a common ground :)
No change requested though, keep it in.

Reply:
I think this is really a matter of taste and personal preference, but I like to have the props directly inside of the corresponding component file, because you'll probably never need them elsewhere. The data, I like to export as a getData function, because than you can simplify your tests and are tightly coupled to it. Most of the times I use a separate "data" folder and a file like "navbarButtons.ts" for this. But you could also argue, that the data file can be placed beside the component.. both is actually ok, I think.